cmd/zed/zed.d/statechange-led.sh

```diff
@@ -1,21 +1,21 @@
 #!/bin/sh
 #
-# Turn off/on the VDEV's enclosure fault LEDs when the pool's state changes.
+# Turn off/on vdevs' enclosure fault LEDs when their pools' state changes.
```

Comment:
The grammar was correct before.  VDEV is an acronym for "Virtual Device" or "Virtual Disk" and acronyms are typically capitalized.  "VDEV's" was also correct since it's the possessive form of the noun.

Reply:
This script toggles enclosure fault LEDs on multiple vdevs at the point when a particular vdev's pool experiences a state change, or "toggles vdevs' enclosure fault LEDs when their pool's state changes"

vdev is not an acronym (that'd be "VD"), but, rather, a shorthand-turned-object-class for "virtual device" as ZFS implements it, consult zpoolconcepts(8), or zpoolprops(8), or zpool-status(8), or